Dockerfile

```diff
@@ -1,14 +1,15 @@
 FROM registry.opensource.zalan.do/stups/alpine:latest
-MAINTAINER Team Teapot @ Zalando SE <team-teapot@zalando.de>
+LABEL maintainer="Team Teapot @ Zalando SE <team-teapot@zalando.de>"
+
+ARG K8S_VERSION=v1.9.5
 
 # install cluster.py dependencies
 # including kubectl
-RUN apk update && \
-    apk add python3 ca-certificates openssl git openssh-client && \
+RUN apk add --no-cache python3 ca-certificates openssl git openssh-client && \
```

Comment:
I see your point about not updating but it's a trade-off between being stable vs. having the latest fixes/security updates (which could break). So far we have not had any problem with this, so I would prefer to leave it as it was.

The other changes are great! Thanks a lot!

Reply:
Sorry, didn't realize `--no-cache` does the same thing.